spec/setup.js

```diff
@@ -16,6 +16,8 @@ global.window.nostoReactTest = true
 
 afterEach(() => {
     window.nosto = undefined
+    window.nostojs = undefined
+    window.Nosto = undefined
```

Comment:
Is this needed?

Reply:
Clearing the state after the tests, it's a good practice to avoid unexpected and hard to debug errors.
If the code we're testing changes the global state (and I assume we're testing global scope objects), we should clear it.

Reply:
yes, it's needed. For Shopify merchants the shopify version of the client script is loaded, which initializes window.Nosto and doesn't rerun if it's already populated